client.js

```diff
@@ -1,41 +1,78 @@
-const WebSocketClient = require("ws");
+const WebSocket = require("ws");
 require("dotenv").config();
 const axios = require("axios").default;
 const fs = require("fs");
 
 //Sending HTTP requests with Axios is as simple as giving an object to the axios() function that contains all of the configuration options and data
 //TODO:set intervals, now it waits and then logs messages like crazy instead having a wait time in between every message
 
-const { APPID_INTAKE, APPID_OUTPUT, APPID_DLQ, URL, TIMEOUT, TOKEN, FILE } =
-  process.env;
+const {
+  APPID_INTAKE,
+  URL,
+  TIMEOUT,
+  FILE,
+  CLIENT_TOKEN,
+  OUTPUT_TOKEN,
+  DLQ_TOKEN,
+  INTAKE_TOKEN,
+} = process.env;
 
 async function main() {
   await axios
-    .get(`${URL}/application/${APPID_INTAKE}/message`, {
+    .get(`${URL}/stream`, {
       timeout: TIMEOUT,
       headers: {
         "Content-Type": "application/json",
-        "X-Gotify-Key": TOKEN,
-        Connection: "Upgrade",
-        Upgrade: "websocket",
+        "X-Gotify-Key": CLIENT_TOKEN,
       },
     })
     .then((response) => {
       const { messages } = response.data;
-      messages.forEach((msg) => {
-        const eachMsg = msg.message;
-        if (!alreadyInFile(eachMsg) && isValidURL(eachMsg)) {
-          fs.appendFileSync(FILE, eachMsg + "\n", { flag: "a+" });
-        } else {
-          handleDLQ(eachMsg);
+      for (const msg of messages) {
+        const msgBody = msg.message;
+        if (!isValidURL(msgBody)) {
+          sendMsg(msgBody, DLQ_TOKEN);
+          deleteMsg(msg.id, CLIENT_TOKEN);
+          continue;
         }
-      });
+        if (!alreadyInFile(msgBody)) {
+          fs.appendFileSync(FILE, msgBody + "\n", { flag: "a+" });
+        }
+        sendMsg(msgBody, OUTPUT_TOKEN);
+        deleteMsg(msg.id, CLIENT_TOKEN);
+      }
     })
-    .catch((err) => {
-      console.log(err.response ? err.response.data : err);
-    });
+    .catch((err) => console.log("CATCH Response error", err.response));
 }
-main();
+// main();
+
+let socket = new WebSocket(`wss://push.4redbuttons.dev/stream`, {
+  headers: { "X-Gotify-Key": CLIENT_TOKEN },
+});
+
+socket.onopen = (event) => {
+  console.log("Websocket connection opened", event.data);
```

Comment:
Add a colon here, too:
```suggestion
  console.log("Websocket connection opened:", event.data);
```

Reply:
Added colon.